src/main/java/seedu/duke/util/Parser.java

```diff
@@ -125,6 +135,10 @@ public Command parseCommand(String userInput) {
         }
     }
 
+    private Command prepareRemind(String userMessage) {
+        return new RemindCommand();
```

Comment:
I just left it in because in the future there might be need to develop looking at future reminders. But then again in the current iteration I saw no need for it / we may not develop it at all cause of limited use cases.